tanner/emulators/template_injection.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+import logging
+
+from urllib.parse import unquote
+from tanner.utils import patterns
+from tanner.utils.aiodocker_helper import AIODockerHelper
+
+
+class TemplateInjection:
+
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.template_injection')
+        self.docker_helper = AIODockerHelper()
+
+    async def get_injection_result(self, payload):
+        execute_result = None
+        github_remote_path = 'https://raw.githubusercontent.com/mushorg/tanner/master/docker/tanner/' \
```

Comment:
better to do it via config value (and probably it's the point when you can provide alternative source)

Reply:
Perfect.